src/plotting/create_plots.py

```diff
@@ -82,9 +82,9 @@ def create_plots(args, root_path, use_datetime=True, test_private_keys: Optional
 
     if args.size < config["min_mainnet_k_size"] and test_private_keys is None:
         log.warning(f"Creating plots with size k={args.size}, which is less than the minimum required for mainnet")
-    if args.size < 22:
-        log.warning("k under 22 is not supported. Increasing k to 22")
-        args.size = 22
+    if args.size < 25:
```

Comment:
I'm not sure if we want to do something with the sizes between 25 and 32 here. I changed it in the `click` interface, but I'm not sure about the need of changing it internally. Actually I'm not 100% sure about the need of changing it here from 22 to 25.

@hoffmang9 what do you think?

Reply:
This is used by the tests so it won't work.
I suggest changing it at the click level. Don't allow less than 32, unless a special flag is passed, like --override-k. What do you think @jespino ?